README.md

```diff
@@ -4,60 +4,143 @@
+SourceCred is organized around a plugin architecture, which ensures that it can
+track and reward any kind of contribution, so long as you can assign addresses
+to contributions, and record how they relate to one another. Currently, we have the following four plugins:
+
+- **GitHub**: Loads GitHub repositories, including issues, pull requests, and reviews
+- **Discourse**: Loads Discourse forums, including posts, topics, and likes
+- **Discord**: Loads Discord servers, including messages and reactions
+- **Initiatives**: Loads manually added contributions. Still in alpha.
+
+Every plugin has a two-part name in the form `$OWNER/$NAME`; for example,
+SourceCred's own GitHub plugin is named `sourcecred/github`.
+
+## The Instance System
+
+SourceCred is organized around the concept of "instances". A SourceCred instance
+contains all of the configuration and data associataed with Cred and Grain, and
+optionally may be set up as a deployable website that displays those scores.
+Each instance has the following directory structure:
+
+```
+./sourcecred.json   # Lists enabled plugins
+./config            # User-edited config files
+./data              # Persistent data, e.g. ledger history
+./output            # Output data, may be removed/regenerated
+./site              # Bundled frontend, if included
+./cache             # Temporary data, should not be checked in to git
+```
 
-## Current Status
+We recommend storing instances in a Git repository. The best way to set up an
+instance is by forking [sourcecred/example-instance].
 
-SourceCred is still in beta; as such, the interfaces are in flux and the documentation is spotty.
-We're working on a polished release which will include more documentation, and more maintainable instances. We expect this to land
-by early July.
+Once your instance is setup, you can update it with the following commands:
 
-For now, if you want to use SourceCred, you might start by forking [MetaGame's Cred Instance].
+- `sourcecred load`: Regenerate the cache
+- `sourcecred graph`: Recompute graphs from cache
+- `sourcecred score`: Re-run Cred calculations
+- `sourcecred site`: Regenerate the website (potentially upgrading it)
+- `sourcecred go`: Runs `load`, `graph` and `score` in sequence.
 
-Note that our next release (v0.7.0) will totally revamp how SourceCred instances are setup, and replace the CLI. As such,
-expect that migrating from v0.6.0 to v0.7.0 will involve making changes to your configuration.
+## Contributing Guidelines
 
-[MetaGame's Cred Instance]: https://github.com/MetaFam/TheSource
+If you'd like to contribute to the codebase, we ask you to follow the following
+steps:
+
+### 1. Drop by [our Discord].
+
+Come to the #intros channel and introduce yourself. Let us know that you're
+interested in helping out. We're friendly and will be happy to help you get
+oriented.
+
+### 2. Read our [Contributing Guidelines].
+
+We pride ourself on tidy software engineering; part of how we do that is by
```

Comment:
This is specific to code contributions. Will some readers come to the README somehow wanting to contribute to something else, think we're not looking for non-code contributions? Perhaps tweak language to convey these are coding contribution guidelines, or point to non-coding contribution guidelines elsewhere also?

Reply:
It actually already said at the top that these instructions are for contributing to the codebase. But I altered the section so it directs everyone to our get-involved page first, and then follows up with explicit code contributions.